cubic-server/command_parser/commands/Time.cpp

```diff
@@ -0,0 +1,153 @@
+#include "Time.hpp"
+#include "Player.hpp"
+#include "Server.hpp"
+#include "World.hpp"
+
+using namespace command_parser;
+
+void Time::autocomplete(std::vector<std::string> &args, Player *invoker) const
+{
+    if (invoker)
+        return;
+    else
+        LINFO("autocomplete time");
+}
+
+int setTimeFromArg(std::string value, int multiplier)
+{
+    int time = 0;
+    try {
+        if (value.back() == 'd' || value.back() == 's' || value.back() == 't')
+            value.pop_back();
+        time = std::stod(value) * multiplier;
+        return time;
+    } catch (const std::exception &e) {
+        LERROR("Expected float");
+        // invoker->sendPlayerChatMessage("Expected float");
+        return -1;
+    }
+}
+
+int setMultiplier(char arg)
+{
+    if (arg == 'd') {
```

Comment:
What is this ?

Reply:
According to https://minecraft.fandom.com/wiki/Commands/time :
![image](https://user-images.githubusercontent.com/62648027/231279662-35748edd-a35f-42ae-b029-a7fb8f1315e8.png)